src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -782,16 +782,16 @@
       <Link>Interop\Windows\Kernel32\Kernel32.cs</Link>
     </Compile>
     <Compile Include="$(CommonSourceRoot)\Interop\Windows\NtDll\CreateDisposition.cs">
-      <Link>Interop\Windows\NtDll\Interop\Windows\NtDll\CreateDisposition.cs</Link>
+      <Link>Interop\Windows\NtDll\CreateDisposition.cs</Link>
     </Compile>
     <Compile Include="$(CommonSourceRoot)\Interop\Windows\NtDll\CreateOptions.cs">
-      <Link>Interop\Windows\NtDll\Interop\Windows\NtDll\CreateOptions.cs</Link>
+      <Link>Interop\Windows\NtDll\CreateOptions.cs</Link>
     </Compile>
     <Compile Include="$(CommonSourceRoot)\Interop\Windows\NtDll\DesiredAccess.cs">
-      <Link>Interop\Windows\NtDll\Interop\Windows\NtDll\DesiredAccess.cs</Link>
+      <Link>Interop\Windows\NtDll\DesiredAccess.cs</Link>
     </Compile>
     <Compile Include="$(CommonSourceRoot)\Interop\Windows\NtDll\FileFullEaInformation.cs">
-      <Link>Interop\Windows\NtDll\Interop\Windows\NtDll\FileFullEaInformation.cs</Link>
+      <Link>Interop\Windows\NtDll\FileFullEaInformation.cs</Link>
     </Compile>
     <Compile Include="$(CommonSourceRoot)\Interop\Windows\NtDll\ImpersonationLevel.cs">
       <Link>Interop\Windows\NtDll\ImpersonaltionLevel.cs</Link>
```

Comment:
```suggestion
      <Link>Interop\Windows\NtDll\ImpersonationLevel.cs</Link>
```

Reply:
It's in impersonal impersonation level, c'mon 😜